accession-commons-monotonic-generator-jpa/src/main/java/uk/ac/ebi/ampt2d/commons/accession/generators/monotonic/MonotonicAccessionGenerator.java

```diff
@@ -105,20 +70,43 @@ private static void assertBlockParametersAreInitialized(ContiguousIdBlockService
         }
     }
 
+    private void recoverState() {
+        if (RUN_RECOVERY && monotonicDatabaseService != null) {
```

Comment:
Should we check that the generator has not SHUTDOWN ?

Reply:
This is a private method and the public method from which this is being called is already doing the check.

Reply:
That makes sense.